src/main/java/matchmaker/backend/controllers/ChallengeController.java

```diff
@@ -64,4 +67,53 @@ public Optional<Challenge> createChallenge(@RequestBody Challenge newChallenge)
         return Optional.of(savedChallenge);
     }
 
-}
+    @GetMapping("/search")
```

Comment:
Dit moet /challenge/search zijn, volgens TO 🤓

Reply:
fixed